### ISSUE TITLE
Always use utf-8 for yaml file parsing

### ISSUE THREAD

**Issue by Avsecz** [kipoi/kipoi#296]
```
(kipoi-DeepSEA__predict) avanti@nandi:/users/avanti/.kipoi/models/DeepSEA/predict$ kipoi predict DeepSEA/predict \
>   --dataloader_args='{'intervals_file': 'example_files/intervals.bed', 'fasta_file': 'example_files/hg38_chr22.fa'}' \
>   -o '/tmp/DeepSEA|predict.example_pred.tsv'
INFO [kipoi.remote] Update /users/avanti/.kipoi/models/
Already up-to-date.
INFO [kipoi.remote] git-lfs pull -I DeepSEA/predict/**
INFO [kipoi.remote] git-lfs pull -I DeepSEA/template/model_files/**                                                               
INFO [kipoi.remote] git-lfs pull -I DeepSEA/template/example_files/**                                                             
INFO [kipoi.remote] git-lfs pull -I DeepSEA/template/**                                                                           
INFO [kipoi.remote] model DeepSEA/predict loaded
Traceback (most recent call last):
  File "/users/avanti/anaconda2/envs/kipoi-DeepSEA__predict/lib/python3.5/site-packages/related/converters.py", line 32, in __call__
    return to_model(self.cls, value)
  File "/users/avanti/anaconda2/envs/kipoi-DeepSEA__predict/lib/python3.5/site-packages/related/functions.py", line 100, in to_model
    value = cls(**value)
  File "<attrs generated init 01422e0d936d1998f14880d701d9f1d1ae5e3c17>", line 4, in __init__
  File "/users/avanti/anaconda2/envs/kipoi-DeepSEA__predict/lib/python3.5/site-packages/kipoi/external/related/converters.py", line 63, in __call__
    return to_model(self.cls, value)
  File "/users/avanti/anaconda2/envs/kipoi-DeepSEA__predict/lib/python3.5/site-packages/related/functions.py", line 100, in to_model
    value = cls(**value)
  File "<attrs generated init cd4eb4961db6f7a600b501d300e8aee6fd3109ea>", line 16, in __init__
  File "/users/avanti/anaconda2/envs/kipoi-DeepSEA__predict/lib/python3.5/site-packages/kipoi/components.py", line 131, in __attrs_post_init__
    object.__setattr__(self, "column_labels", [l.rstrip() for l in ifh])
  File "/users/avanti/anaconda2/envs/kipoi-DeepSEA__predict/lib/python3.5/site-packages/kipoi/components.py", line 131, in <listcomp>
    object.__setattr__(self, "column_labels", [l.rstrip() for l in ifh])
  File "/users/avanti/anaconda2/envs/kipoi-DeepSEA__predict/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 7664: ordinal not in range(128)
```

This error went away when using:
```
export LANG=en_US.UTF-8
export LC_CTYPE=en_US.UTF-8
```

**Comment by Avsecz** [comment]
Forced UTF-8 based decoding when reading yaml files here: https://github.com/kipoi/kipoi/pull/377

**Comment by Licko0909** [comment]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 7664: ordinal not in range(128)

I try the answer of it, but it didn't work
![image](https://user-images.githubusercontent.com/27897166/56570801-2f32f000-65ee-11e9-9e49-7a58b8e9a45a.png)

**Comment by Avsecz** [comment]
Hm. Which version of Kipoi are you using?

**Comment by Licko0909** [comment]
> Hm. Which version of Kipoi are you using?

 It's kipoi==0.6.9 in the Docker container。

![image](https://user-images.githubusercontent.com/27897166/56625590-998c7480-6670-11e9-87b3-dcb162673833.png)

![image](https://user-images.githubusercontent.com/27897166/56625608-aa3cea80-6670-11e9-8ddf-3cb841e83bdf.png)

**Comment by Avsecz** [comment]
What docker container are you using? You could try using https://cloud.docker.com/u/kipoi/repository/docker/kipoi/models

**Comment by Licko0909** [comment]
> What docker container are you using? You could try using https://cloud.docker.com/u/kipoi/repository/docker/kipoi/models


Thank you for your recommendation. I have tried the link you gave before. He does not support the GPU version. So I chose to use NVIDIA's container.  [Nvidia cuda 9.0 docker container](https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/9.0/runtime/cudnn7/Dockerfile)


 But there is a problem: UnicodeDecodeError

**Comment by Avsecz** [comment]
Can you paste the exact commands you executed so that we can reproduce the bug

**Comment by Avsecz** [comment]
Can you try setting the following environment variable in the container:

```
export LC_ALL=C
```

**Comment by Licko0909** [comment]
> Can you try setting the following environment variable in the container:
> 
> ```
> export LC_ALL=C
> ```
Thank you, but it not work.

![image](https://user-images.githubusercontent.com/27897166/56714425-552fd000-6767-11e9-9b3c-7cd3fc355dc4.png)

**Comment by Licko0909** [comment]
> Can you paste the exact commands you executed so that we can reproduce the bug?

**Thank you very much first! Here is my process**, if necessary, I will upload my image to `hub.docker`

**1、I first pulled the corresponding image of nvidia.**
`docker pull nvidia/cuda:9.0-cudnn7-runtime`

**2、Then I tested it to make sure I can use gpu**
`docker run --runtime=nvidia --rm nvidia/cuda:9.0-cudnn7-runtime nvidia-smi`

3、Then I install according to your prompts until `kipoi ls`
![image](https://user-images.githubusercontent.com/27897166/56714832-91176500-6768-11e9-9ffb-81b2a53f2205.png)

```
bash /home/bio//Miniconda3-4.4.10-Linux-x86_64.sh
conda config --add channels https://mirrors.ustc.edu.cn/anaconda/pkgs/free/
conda config --set show_channel_urls yes
source ~/.bashrc

conda install -y -c conda-forge jupyterlab
conda install -y nb_conda datrie
pip install numpy pandas matplotlib seaborn snakemake kipoi>=0.6.0   
apt-get install git
kipoi ls
```

**Comment by DerThorsten** [comment]
> > What docker container are you using? You could try using https://cloud.docker.com/u/kipoi/repository/docker/kipoi/models
> 
> Thank you for your recommendation. I have tried the link you gave before. He does not support the GPU version. So I chose to use NVIDIA's container. [Nvidia cuda 9.0 docker container](https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/9.0/runtime/cudnn7/Dockerfile)

Actually the docker image has gpu support.
The dockerfile is here: https://github.com/DerThorsten/kipoi/tree/docker/example/models/shared/envs/docker-gpu
The build docker cotainer with gpu support is here
https://cloud.docker.com/repository/docker/derthorsten/kipoi-test the tag is 'latest-gpu'

**Comment by Licko0909** [comment]
> > > What docker container are you using? You could try using https://cloud.docker.com/u/kipoi/repository/docker/kipoi/models
> > 
> > 
> > Thank you for your recommendation. I have tried the link you gave before. He does not support the GPU version. So I chose to use NVIDIA's container. [Nvidia cuda 9.0 docker container](https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/9.0/runtime/cudnn7/Dockerfile)
> 
> Actually the docker image has gpu support.
> The dockerfile is here: https://github.com/DerThorsten/kipoi/tree/docker/example/models/shared/envs/docker-gpu
> The build docker cotainer with gpu support is here
> https://cloud.docker.com/repository/docker/derthorsten/kipoi-test the tag is 'latest-gpu'

Thank you very much! I will try it soon!

**Comment by DerThorsten** [comment]
Hi, @Licko0909 
I could reproduce your error and I think I found the problem.
In the file ` $HOME\.kipoi/models/rbp_eclip/model-template.yaml ` there are non ascii chars in  
`Žiga Avsec` 
You can change the file by hand or wait until we merged https://github.com/kipoi/models/pull/173

**Comment by Avsecz** [comment]
:| (sorry about that) Thanks @DerThorsten!

**Comment by Licko0909** [comment]
> Hi, @Licko0909
> I could reproduce your error and I think I found the problem.
> In the file `$HOME/.kipoi/models/rbp_eclip/model-template.yaml` there are non ascii chars in
> `Žiga Avsec`
> You can change the file by hand or wait until we merged [kipoi/models#173](https://github.com/kipoi/models/pull/173)

Wow! So cool! Thanks!@Avsecz @DerThorsten
